### PR TITLE
feat: add new blog - git branching overiew

### PR DESCRIPTION
**Pull request by achag033**
## Summary

This commits add new blog for git branching overview - help give clear idea and set of commands for better collaborative experience

## Basic example

Include a basic example, screenshots, or links.

## Motivation

Why are we doing this? What use cases does it support? What is the expected outcome?

## Checks

- [ ] Read [Create a Pull Request](https://getdoks.org/docs/contributing/how-to-contribute/#create-a-pull-request)
- [ ] Supports all screen sizes (if relevant)
- [ ] Supports both light and dark mode (if relevant)
- [ ] Passes `npm run test`

### CODE REVIEW

File: archetypes/docs/_index.md [hunk 1]
```diff
@@ -7,5 +7,4 @@ lastmod: 2022-01-25T14:40:56+01:00
 draft: false
 images: []
 type: docs
-robots: "noindex"
```
Comment (anishbista60) [inline archetypes/docs/_index.md:10]: Seems , you have reverted the changes to stop failure.

**Comment by anishbista60** [comment]
@achag033 Thank you. LGTM. I don't have permission to merge it. @satrox28 please take a look.

**Comment by achag033** [comment]
> @achag033 Thank you. LGTM. I don't have permission to merge it. @satrox28 please take a look.

will make more changes on it, will merge after that.

**Comment by anishbista60** [comment]
@achag033 In term of best practices, IMO it would be to have a single commit in single work . Instead of adding commit in every changes , it could be better to amend the changes to previous commit using `git commit --amend`.

**Comment by achag033** [comment]
> @achag033 In term of best practices, IMO it would be to have a single commit in single work . Instead of adding commit in every changes , it could be better to amend the changes to previous commit using `git commit --amend`.

**Comment by achag033** [comment]
> > @achag033 In term of best practices, IMO it would be to have a single commit in single work . Instead of adding commit in every changes , it could be better to amend the changes to previous commit using `git commit --amend`.

Agree with that yes

**Comment by anishbista60** [comment]
And it would be better to squash these four commit into single one cause at the end it would  make everyone easy to get actual gist about the PR

Thankyou ! Rest blog is pretty good.

**Pull request merged**